spec/models/mission_spec.rb

```diff
@@ -52,4 +61,36 @@
       it { is_expected.to have_and_belong_to_many(:addresses) }
     end
   end
+
+  describe '#selectable_time_slots' do
+    it 'returns the time slots with at least one slot available' do
+      mission = create :mission, genre: 'regulated'
+
+      response = mission.selectable_time_slots
```

Comment:
do not use `response` as the variable name, as it can shadow the variable `response` set by RSpec in some situations (e.g. in request tests, when you make a `get` request, RSpec creates a `response` variable that you can access to inspect what was the response of the request)

It shouldn't cause any problem in a unit test (for this example here, there is no problem), but I think it's better to avoid to keep a good habit.

FYI: other variables names you should avoid in RSpec for the same reasons: `request`, `subject` (there might be other, but I don't remember them right now)

Reply:
ok ty